config/admin-functions/basic-configuration.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+: "${VX_FUNCTIONS_ROOT:="$(dirname "$0")"}"
+: "${VX_CONFIG_ROOT:="/vx/config"}"
+
+source "${VX_FUNCTIONS_ROOT}/../read-vx-machine-config.sh"
+clear
+
+echo -e "\e[1mWelcome to Basic Configuration\e[0m"
+echo "You're going to do great"
+
+echo
+echo -e "\e[1mStep 1: Set Machine ID\e[0m"
+${VX_FUNCTIONS_ROOT}/choose-vx-machine-id.sh
+
+echo
+echo -e "\e[1mStep 2: Set Clock\e[0m"
+${VX_FUNCTIONS_ROOT}/set-clock.sh
+
+echo
+echo -e "\e[1mStep 3: Record Machine Metadata\e[0m"
+echo 'Setting up TOTP...'
+TOTP_URI=`${VX_FUNCTIONS_ROOT}/reset-totp.sh | grep otpauth`
```

Comment:
I generally prefer `$(…)` over backticks because it's possible to nest `$(…)`. Up to you, though.